src/bowlingApp.js

```diff
@@ -0,0 +1,132 @@
+$(document).ready(function() {
+
+  var bowlingScorer = new BowlingScorer();
+
+  $("#score").text(bowlingScorer.returnScore());
+  // $("#frame_" + )
+
+  $("#add_score_0").click(function() {
```

Comment:
You have a massive amount of duplicated code in this file. It wouldn't take much effort to extract the duplication into a helper function and call it from each click function, passing in the number of the frame.